tiny_dnn/core/kernels/global_avepool_op.h

```diff
@@ -0,0 +1,37 @@
+/*
+    Copyright (c) 2017, Taiga Nomi
+    All rights reserved.
+
+    Use of this source code is governed by a BSD-style license that can be found
+    in the LICENSE file.
+*/
+#pragma once
+
+#include "tiny_dnn/core/framework/op_kernel.h"
+#include "tiny_dnn/core/kernels/global_avepool_op_internal.h"
+
+namespace tiny_dnn {
+
+class GlobalAvePoolOp : public core::OpKernel {
```

Comment:
at some point I had some class naming conflicts that I could solve by using `CamelCase` then we came with this convention stuff